B827EBFFFE481662.json

```diff
@@ -0,0 +1,18 @@
++{
+ +  "gateway_conf": {
```

Comment:
Please check the content of the file, it seems you copy & pasted a diff and the + signs at the beginning of each line stayed there, making it an invalid json file.